yb-voyager/src/tgtdb/postgres.go

```diff
@@ -559,6 +574,10 @@ func (pg *TargetPostgreSQL) ExecuteBatch(migrationUUID uuid.UUID, batch *EventBa
 			return false, fmt.Errorf("failed to commit transaction : %w", err)
 		}
 
+		if discrepancyFoundInBatch(batch, rowsAffectedInserts, rowsAffectedDeletes, rowsAffectedUpdates) {
+			log.Infof("Committed batch(%s): batch-size %d rowsAffectedInserts=%d, rowsAffectedInserts=%d rowsAffectedInserts=%d", batch.ID(), len(batch.Events), rowsAffectedInserts, rowsAffectedDeletes, rowsAffectedUpdates)
```

Comment:
WARN level?

Reply:
Also
- mention discrepancy in log `Discrepancy in committed batch` 
- list the batch.inserts, batch.updates, batch.deletes
- log msg has "rowsAffectedInserts" for all types"

Reply:
pls take care of this in other target dbs as well. 
Perhaps, you could create a common func `reportDiscrepancyInEventBatchIfAny` that checks for discrepancy and logs

Reply:
For now, I have added all the vsns in the batch in this Discrepancy log itself. We can think of logging vsns other cases later.

Reply:
Pipeline for live migration tests and oracle test -
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/2423/